From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Tue, 14 May 2019 15:57:19 -0700
Subject: [PATCH] Make the emacs compiled without X11 call the one compiled
 with

Signed-off-by: Thiago Macieira <thiago.macieira@intel.com>
---
 src/emacs.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/emacs.c b/src/emacs.c
index 0a90b09..e48af2a 100644
--- a/src/emacs.c
+++ b/src/emacs.c
@@ -2008,6 +2008,12 @@ Using an Emacs configured with --with-x-toolkit=lucid does not have this problem
 
     skip_args = count_before;
   }
+
+  if (!noninteractive && !inhibit_window_system)
+  {
+    /* attempt to launch the X11-capable version, if it is present */
+    (void) execvp ("emacs-x11", argv);
+  }
 #endif
 
   /* argmatch must not be used after here,
