From 26dd4bc1ab3f4d347d8179316dd3bdeb26cfc41d Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 12 Nov 2024 17:03:04 -0800
Subject: [PATCH] Don't fingerprint the pdmp file

---
 Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 996f7b8..da28054 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -334,7 +334,7 @@ unexport EMACSDATA EMACSDOC EMACSLOADPATH EMACSPATH
 EMACS_NAME = `echo emacs | sed '$(TRANSFORM)'`
 EMACS = ${EMACS_NAME}${EXEEXT}
 EMACSFULL = `echo emacs-${version} | sed '$(TRANSFORM)'`${EXEEXT}
-EMACS_PDMP = `./src/emacs${EXEEXT} --fingerprint`.pdmp
+EMACS_PDMP = ${version}.pdmp
 
 # Subdirectories to make recursively.
 SUBDIR = $(NTDIR) lib lib-src src lisp
-- 
2.47.0

